openapi.yaml

```
@@ -2250,6 +2250,12 @@ paths:
               description: MAY include the costs for processing and downloading the data.
               schema:
                 $ref: '#/components/schemas/money'
+            OpenEO-Identifier:
+              description: Identifier associated with the synchronous processing request.
```

Should we indicate in the description that it's optional?

```suggestion
              description: Optionally, an identifier associated with the synchronous processing request.
```